app/src/main/java/umc/cozymate/ui/cozy_bot/CozyBotFragment.kt

```
@@ -67,14 +69,22 @@ class CozyBotFragment : Fragment() {
+                val intent =
+                    Intent(requireActivity(), OwnerRoomDetailInfoActivity::class.java).apply {
+                        putExtra(OwnerRoomDetailInfoActivity.ARG_ROOM_ID, roomId)
+                    }
                 startActivity(intent)
             }
             // 초기 룸로그 로드
             viewLifecycleOwner.lifecycleScope.launch {
-                viewModel.getRoomInfoById()
+                try {
+                    viewModel.getRoomInfoById()
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)